CHANGES.md

```diff
@@ -14,6 +14,9 @@ Unreleased
 - Add `dune build --dump-gc-stats FILE` argument to dump Garbage Collection
   stats to a named file. (#8072, @Alizter)
 
+- Add `(shell ..)` option to cram test stanza and user action specification.
```

Comment:
```suggestion
- Add `(shell ..)` option to `(cram)` stanza and user action specification.
```

Reply:
addressed in 2a23c30773325a1bf6086f3bfe0678009535603e